rsconf/component/sirepo.py

```diff
@@ -187,6 +187,9 @@ def _install_dirs_and_files(self):
             self.install_directory(p.join(c))
         self.install_access(mode='400')
         for c in z.feature_config.proprietary_sim_types:
+            # TODO(e-carlin): Need a better way to configure this
+            if c == 'jupyterhublogin':
```

Comment:
@robnagler how should we ask the question of whether or not a proprietary code has an rpm?

Reply:
I thought I saw you called proprietary_rpm or something like that. Any interface is fine as it is quite internal.